src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final int BOUND = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        int num = random.nextInt(BOUND);
+        Ball ball = new Ball(color, num);
+        return ball;

```

Comment:
in such cases, don't store to a local variable, use expression right away
```suggestion
        return new Ball(color, num);
```